scripts/bootstrap.sh

```
@@ -0,0 +1,50 @@
+# scripts/bootstrap.sh — macOS/Linux dev machine setup
+# macOS: run `brew bundle`
+# Linux: run `sudo apt-get install -y jq shellcheck gh curl git`
+# then `source scripts/bootstrap.sh`
+
+#!/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)